lib/rucio/core/request.py

```diff
@@ -1835,6 +1835,7 @@ def get_source_rse(request_id, src_url, session=None, logger=logging.log):
             return None, None
 
         sources = get_sources(request_id, session=session)
```

Comment:
why not 

`sources = get_sources(request_id, session=session) or []` ?

Reply:
are we paying by line of code ? like in the good old "C compilation by line of code" days ? :) 

In reality, I initially did it like that. After that I changed because I considered it's much more explicit this way and less likely to be missed by somebody reading the code.

Reply:
haha :-D

och, I don't mind much I was just interested